.github/workflows/e2e-tests.yml

```diff
@@ -5,9 +5,6 @@ on:
     branches: [master]
   pull_request:
     branches: ['**']
-    paths-ignore:
-      - '**/*.md'
-      - '**/*.MD'
```

Comment:
These weren't working as expected, will need to try out a new approach in a later PR.